schema/project.js

```diff
@@ -192,6 +228,19 @@ class Project extends BaseModel {
           to: 'establishments.id'
         }
       },
+      establishments: {
```

Comment:
Could we call this `additionalEstablishments`? It feels confusing to have `establishment` and `establishments` mean slightly different things.

Reply:
yup 👍